go.mod

```diff
@@ -1,6 +1,8 @@
 module github.com/docker/buildx
 
-go 1.21
+go 1.21.0
+
+toolchain go1.22.5
```

Comment:
😠

Reply:
you can safely remove `toolchain go1.22.5`

Reply:
ah nevermind you already removed it